testsuite/integration/elytron-oidc-client/src/test/resources/org/wildfly/test/integration/elytron/oidc/client/logout/BackChannelLogoutApp-oidc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "realm" : "WildFly",
+    "resource" : "BackChannelLogoutApp",
+    "public-client" : "false",
+    "auth-server-url" : "${logout.auth.server.url}",
```

Comment:
It would be good to update the .json files to use provider-url and client-id instead of realm, resource, and auth-serverl-url (the latter three are the old Keycloak-specific way, and the first two are the more generic options that work for all providers).

Reply:
done